server_admin/topics/users/proc-setting-password-user.adoc

```diff
@@ -19,8 +19,13 @@ If a user already has a password, it can be reset in the *Reset Password* sectio
 +
 NOTE: If *Temporary* is *ON*, the user must change the password at the first login. To allow users to keep the password supplied, set *Temporary* to *OFF.*  The user must click *Set Password* to change the password.
```

Comment:
```suggestion
NOTE: If *Temporary* is *ON*, the user must change the password at the first login. To allow users to keep the password supplied, set *Temporary* to *OFF.*  
```

Reply:
Also, line 17 on the procedure above does not apply to the new UI:  
. Type a new password in the *Set Password* section.
The procedure is different.

I think you could use the user-credentials.png here.
 image:{project_images}/user-credentials.png[user credentials]